airbyte-integrations/connectors/source-gitlab/metadata.yaml

```diff
@@ -29,6 +29,21 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      4.0.0:
+        message:
+          In this release, a format of stream state has been changed due to migration to low-code.
+          Users will need to reset the affected streams after upgrading.
+        upgradeDeadline: "2024-04-08"
+        scopedImpact:
+          - scopeType: stream
+            impactedScopes:
+              - "commits"
+              - "issues"
+              - "merge_requests"
+              - "merge_request_commits"
+              - "pipelines"
+              - "pipelines_extended"
+              - "jobs"
```

Comment:
This stream is not incremental. Why would its state format change? Or why is it in the impactedScope?

Reply:
Removed, thanks!